### PR TITLE
Use targeted refresh during transformation

### PR DESCRIPTION
**Pull request by fdupont-redhat**
Previously, during transformation, we triggered a full refresh to be sure that the VM was in the inventory for the post migration steps. But, this consumed a lot of resources compared to the single VM we want to track. And on a huge inventory with thousands of VMs, triggering a full refresh for each migration can soon be problematic.

This PR adds a new instance in the event switchboard that triggers a targeted refresh for the associated VM. The event is named `VM_IMPORT_FROM_CONFIGURATION_EXECUTED_SUCCESSFULLY`.

Associated RHBZ: https://bugzilla.redhat.com/show_bug.cgi?id=1607368

### CODE REVIEW

**Comment by fdupont-redhat** [comment]
@miq-bot add-label transformation
@miq-bot add-label enhancement
@miq-bot add-label gaprindashvili/yes
@miq-bot add-label blocker

File: content/automate/ManageIQ/Transformation/Infrastructure/VM/rhevm.class/__methods__/checkvmininventory.rb [hunk 1]
```diff
@@ -16,27 +16,13 @@ def main
                 destination_vm = ManageIQ::Automate::Transformation::Infrastructure::VM::RedHat::Utils.new(destination_ems).vm_find_by_name(source_vm.name)
                 raise "VM #{source_vm.name} not found in destination provider #{destination_ems.name}" if destination_vm.nil?
 
-                finished = false
-
                 destination_vm_vmdb = @handle.vmdb(:vm).where(["ems_ref = ?", destination_vm.href.gsub(/^\/ovirt-engine/, '')]).first
```
Comment (gmcculloug) [inline content/automate/ManageIQ/Transformation/Infrastructure/VM/rhevm.class/__methods__/checkvmininventory.rb]: Realize this is outside of the changes in this PR but future refactorings should be:

```ruby
destination_vm_vmdb = @handle.vmdb(:vm).find_by(:ems_ref => destination_vm.href.gsub(/^\/ovirt-engine/, ''))
```

Comment (fdupont-redhat) [reply content/automate/ManageIQ/Transformation/Infrastructure/VM/rhevm.class/__methods__/checkvmininventory.rb]: Changed. I suppose it has better performance.

**Comment by simaishi** [comment]
Gaprindashvili backport details:

```
$ git log -1
commit f559300aad804be604f5b8bc3f5f9fd0a22e36dd (HEAD -> gaprindashvili, origin/gaprindashvili)
Author: Greg McCullough <gmccullo@redhat.com>
Date:   Mon Jul 30 15:33:29 2018 -0400

    Merge pull request #381 from fdupont-redhat/v2v_use_targeted_refresh
    
    Use targeted refresh during transformation
    (cherry picked from commit 4a918271d51c80cfc56941501754249b9ef1767a)
    
    https://bugzilla.redhat.com/show_bug.cgi?id=1610550
```

**Pull request merged**